docs/data/material/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -194,6 +194,31 @@ Modify your `package.json` commands:
   }
 ```
 
+If you are using Next.js >= v12.1.1, you can make use of their [Modularize Imports](https://nextjs.org/docs/advanced-features/compiler#modularize-imports) feature.
```

Comment:
This section of the docs starts to be quite long. I think that it could be good to change the headers to help navigate the content.